Applications/DataExplorer/DataView/ProcessModel.cpp

```diff
@@ -0,0 +1,179 @@
+/**
+ * \file
+ * \author Karsten Rink
+ * \date   2010-10-18
+ * \brief  Implementation of the ProcessModel class.
+ *
+ * \copyright
+ * Copyright (c) 2013, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "ProcessModel.h"
+
+// ThirdParty/logog
+#include "logog/include/logog.hpp"
+
+// ** INCLUDES **
+#include "Applications/DataHolderLib/FemCondition.h"
+#include "GeoLib/GEOObjects.h"
+#include "GeoLib/GeoObject.h"
+
+#include "CondItem.h"
+#include "GeoType.h"
+#include "ProcessVarItem.h"
+
+#include <vtkPolyDataAlgorithm.h>
+#include <QFileInfo>
```

Comment:
it is custom to include the system headers after the "ProcessModel.h"
logog is also a system header...